lib/mpl_toolkits/axisartist/axislines.py

```diff
@@ -108,41 +108,43 @@ def update_lim(self, axes):
         delta2 = _api.deprecated("3.6")(
             property(lambda self: 0.00001, lambda self, value: None))
 
+        def _to_xy(self, *, variable, fixed):
```

Comment:
It's good to make that concept explicit instead of wildly juggling indices in the code. Still, I find this quite hard to follow. Can we move this up one level in the semantics:

```
def _to_xy(self, values, const):
    """
    Create a (len(values), 2)-shape array representing (x, y) pairs.

    *values* go into the coordinate determined by ``self.nth_coord``.
    The other coordinate is filled with the constant *const*.

    Example::

        >>> self.nth_coord = 0
        >>> self._to_xy([1, 2, 3], const=0)
        array([[1, 0],
               [2, 0],
               [3, 0]])
    """

Reply:
Yes, that looks like a good description.